src/mod_maxminddb.c

```diff
@@ -153,6 +166,16 @@ module AP_MODULE_DECLARE_DATA maxminddb_module = {
     maxminddb_register_hooks /* register hooks                      */
 };
 
+static void maxminddb_kv_set(maxminddb_config *conf,
```

Comment:
There's a call to apr_table_set() in set_network_environment_variable() - should we use this function there too?

Reply:
Good call. I missed that on rebase.